python/packages/zarathustra/agents/oracle_verifier/aea-config.yaml

```diff
@@ -121,10 +121,25 @@ models:
       - eightballer/oracle_verifier_chronicle
       - eightballer/oracle_verifier_pyth
       - eightballer/oracle_verifier_flare
+      subgraph_sync_tolerance: 3
       subgraph_api_key: ${GRAPH_API_KEY:str}
-      subgraph_url: https://gateway-arbitrum.network.thegraph.com/api/{api_key}/subgraphs/id/EYCKATKGBKLWvSfwvBjzfCBmGwYNdVkduYXVivCsLRFu
-      subgraph_query: '{ uniswapFactories {id pairCount totalVolumeUSD totalVolumeETH
-        } tokens { id symbol name decimals } }'
+      subgraph_queries:
+        uniswap_v2_arbitrum:
+          url: https://gateway-arbitrum.network.thegraph.com/api/{api_key}/subgraphs/id/EYCKATKGBKLWvSfwvBjzfCBmGwYNdVkduYXVivCsLRFu
+          chain: Arbitrum
+          queries:
+            - '{
+                _meta { block { number } }
+                tokens { id symbol name decimals }
+              }'
```

Comment:
1. you basically always want to query the block number alongside the data so that these are associated. We could leave this up to the user to define as aprt of the (list of) query (/ies)
2. I did not figure out a nice way to combine a list of queries into a single query. 

We could use a library for graphql handling, but rather not introduce a dependency, then again if the alternative is dirty string parsing I would certainly consider it


i.e. if you have these two queries


- `{ _meta { block { number } } }`
- `{ tokens { id symbol name decimals }`

you effectively need to remove outer brackets and wrap them together in a new pair of brackets

```
{
  _meta { block { number } }
  tokens { id symbol name decimals }
}
```